src/autoscaler/integration/integration_suite_test.go

```diff
@@ -65,6 +64,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	if e != nil {
 		Fail("can not connect database: " + e.Error())
 	}
+	clearDatabase()
```

Comment:
This should be done above. This is done on every parallel instance rather than on instance 0. We should only clear once.